kubejs/server_scripts/gregtech/reconstruction.js

```diff
@@ -107,4 +107,36 @@ ServerEvents.recipes(event => {
             .duration(80)
             .EUt(56)
     })
+
+    // Flower cycle convenience recipes
+    const flowerCycle = [
+        'dandelion',
+        'poppy',
+        'blue_orchid',
+        'allium',
+        'azure_bluet',
+        'red_tulip',
+        'orange_tulip',
+        'white_tulip',
+        'pink_tulip',
+        'oxeye_daisy',
+        'cornflower',
+        'lily_of_the_valley',
+        //'sunflower',  //Flowers that give 2x as much dye are excluded. Sorry!
+        //'lilac',
+        //'rose_bush',
+        //'peony',
+        'spore_blossom',
+        'wither_rose',
```

Comment:
This already has a reconstructor recipe (dried kelp). And kelp was available from plant balls (probably accidentally removed in #1174). So it might make sense to exclude it.

Additionally, it would probably make sense to add cactus to this list.

Reply:
Maybe.
It might also just make more sense to scrap the flower cycle and have NM players just buy flowers with coins like they can with Cocoa Beans and Cactus - HM/Expert players can do the exploring.

@ThePansmith what do you think?

Reply:
Would be a lot of recipe patterns we would need to make sure don't overlap with preexisting ones. Given that its just dyes, its fine to be an AR recipe.